README.md

```diff
@@ -39,15 +39,15 @@ Integrations are also provided with:
 
 ### Caching
 
-In order to prevent a call to be made each time a signing key needs to be retrieved you can also configure a cache as follows. If a signing key matching the `kid` is found, this will be cached and the next time this `kid` is requested the signing key will be served from the cache instead of calling back to the JWKS endpoint.
+In order to prevent a call to be made each time a signing key needs to be retrieved a cache is implemented. If a signing key matching the `kid` is found, this will be cached and the next time this `kid` is requested the signing key will be served from the cache instead of calling back to the JWKS endpoint.
```

Comment:
The first sentence reads a bit awkwardly. Perhaps something like `By default, signing keys are cached to prevent...`.